template/package.json

```
@@ -4,7 +4,10 @@
   "private": true,
   "scripts": {
     "start": "react-native start",
-    "test": "jest"
+    "run-android": "react-native run-android",
+    "run-ios": "react-native run-ios",
+    "test": "jest",
+    "postinstall": "jetify"
```

why not:
```suggestion
    "postinstall": "npx jetifier"
```
so it's easier to remove once everything is migrated?

---

@thymikee is `npx` needed? `npm` will run binaries under node_modules fine

---

`npx` would avoid adding `jetifier` as a dependency

---

@thymikee yea but we already add it as a dep, and when the user removes it from dep, they can remove the postinstall script too. imo it doesn't change anything

---

It's just my preference, but yea, may feel weird for some.